src/screens/ProfileScreen/index.js

```diff
@@ -44,10 +38,9 @@ import useProfileScreenHook, {
   TAB_INDEX_ANONYMOUS,
   TAB_INDEX_SIGNED
 } from '../../hooks/screen/useProfileScreenHook';
+import {useAfterInteractions} from '../../hooks/useAfterInteractions';
 import {Analytics} from '../../libraries/analytics/firebaseAnalytics';
-import {ButtonNewPost} from '../../components/Button';
-import {Context} from '../../context';
-import {DEFAULT_PROFILE_PIC_PATH, SOURCE_MY_PROFILE} from '../../utils/constants';
+import {deleteAnonymousPost, deletePost} from '../../service/post';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import order has been changed. Ensure that these changes do not break any dependencies in the code.
```diff
- import {Analytics} from '../../libraries/analytics/firebaseAnalytics';
- import {ButtonNewPost} from '../../components/Button';
- import {Context} from '../../context';
- import {DEFAULT_PROFILE_PIC_PATH, SOURCE_MY_PROFILE} from '../../utils/constants';
+ import {useAfterInteractions} from '../../hooks/useAfterInteractions';
+ import {Analytics} from '../../libraries/analytics/firebaseAnalytics';
+ import {deleteAnonymousPost, deletePost} from '../../service/post';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->